docs/pages/enterprise/workflow/index.mdx

```diff
@@ -1,281 +1,48 @@
 ---
-title: Access Requests for Infrastructure Access
-description: Teleport allows users to request new roles with elevated privileges from the CLI or UI. Requests can be escalated via ChatOps or anywhere else via our flexible Authorization Workflow API.
-h1: Teleport Access Requests
+title: Just-in-time Access Requests
+description: Teleport allows users to request new access capabilities from the CLI or UI. Requests can be escalated via ChatOps or anywhere else via our flexible Authorization Workflow API.
+h1: Teleport Just-in-time Access Requests
```

Comment:
Do we need the title and the h1 here? What does it look like rendered?

Reply:
The title does not get rendered on the page, it shows up in the tab header as `Just-in-time Access Requests | Teleport Docs`. The current title seems to show up in google results as `Access Requests for SSH and Kubernetes Access - Teleport`. So it seems a `Teleport` suffix is always added. Here's the current page https://goteleport.com/docs/enterprise/workflow/

Here's a screenshot of the new index page:
<img width="1790" alt="Screen Shot 2022-06-24 at 10 02 37 AM" src="https://user-images.githubusercontent.com/16807901/175609425-02cb23ca-0c33-4449-aecd-0497e9a9fdb2.png">

One of the Resource Access Requests page:
<img width="1790" alt="Screen Shot 2022-06-24 at 10 11 32 AM" src="https://user-images.githubusercontent.com/16807901/175609838-b5892d51-4d0d-4172-8a09-d5a449b00e3e.png">

And one of the Role Access Requests page:
<img width="1790" alt="Screen Shot 2022-06-24 at 10 11 49 AM" src="https://user-images.githubusercontent.com/16807901/175609894-401c0278-65df-4426-9081-0636b7355561.png">